README.md

```diff
@@ -62,21 +63,13 @@ Simply build and run the container:
 8. Run the bot:
```

Comment:
Can you update the link to the guide to invite the bot to the [official documentation](https://discordpy.readthedocs.io/en/stable/discord.html)?

Reply:
yup, done :)